go/master/service.go

```diff
@@ -123,6 +214,8 @@ func (s *Service) GetTask(dummy int, task *Task) error {
 		return err
 	}
 
+	*task = t.Task
+
```

Comment:
How about change type of `taskQueues.Failed` from `[]Task` to `[]taskEntry`, so that we can use `numTimeout` to record the failed times.